content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-validator.md

```diff
@@ -118,32 +118,104 @@ On your validator:
+
+This configuration connects your validator to the XRP Ledger network using discovered peers.
+
+Specifically, it instructs your validator to connect to one of two public hubs, `r.ripple.com 51235` or `zaphod.alloy.ee 51235`. Once connected, the public hub suggests other servers that might be looking for peers and provides your validator with contact information it can use to connect to them. Your validator can then connect to these servers and ask them for contact information for other servers until your validator has enough reliable peers that you don't have to worry about one or two of them suddenly dropping offline.
+
+Your validator needs to connect to the public hub _once_ and only for a short amount of time to bootstrap its other peer connections. Your validator saves the addresses of these other peers so it can try reconnecting directly to those peers later, even if you restart your validator.
+
+**Pros:**
+
+  - Increases the chances of your validator finding a peer, probably several, to connect to.
+  - Creates the opportunity for a lot of direct peer connections. Being able to request content from more peers enables your validator to parallelize the process of downloading the current ledger (when syncing) or backfilling history and gives it direct access to a wider selection of history.
+  - Lowers the possibility of your validator disconnecting from the network because it provides the ability to constantly replace disconnected peers with new ones.
+
+**Cons:**
+
+  - Doesn't allow you to select your validator's peers, which means that you have no idea whether your peers may decide to act maliciously. If they do, your server has some protections that it can use to [try to detect](transaction-censorship-detection.html) and ignore or block malicious peers. ***TODO: what are the protections that enable your server to block or ignore malicious peers? The consensus process itself? https://developers.ripple.com/consensus-protections.html***
+  - May connect you to peers that disconnect and change frequently, depending on whether you manage to connect to any reliable ones.
+
+_**To connect your validator to the XRP Ledger network using discovered peers,**_ do not provide values for the `[ips]`, `[ips_fixed]`, and `[peer_private]` stanzas in your validator's `rippled.cfg` file. The [example rippled.cfg file](https://github.com/ripple/rippled/blob/develop/cfg/rippled-example.cfg) is delivered with this configuration.
+
+
+### Connect using proxies
+
+This configuration connects your validator to the network through stock `rippled` servers that you run yourself. These proxy servers sit between your validator and inbound and outbound network traffic.
+
+**Note:** While these servers are acting as proxies, they are not web proxies for HTTP(S) traffic.
+
+**Pros:**
+
+  - Provides your validator with connections to the network that are as redundant and reliable as you make them because you run the peer proxy servers yourself.
```

Comment:
Another (related) point worth making:
- You can guarantee that these servers will try to maintain a connection with your validator.

(That's the biggest difference from relying on public hubs, which don't know who your validator is because they don't have you in their `[ips_fixed]`.)